postproengine/spod.py

```diff
@@ -435,6 +433,7 @@ def execute(self, verbose=False):
 
                 if compute_eigen_vectors:
                     self.POD_modes       = {corr: 0 for corr in correlations}
+                    self.POD_proj_coeff  = {corr: 0 for corr in correlations}
```

Comment:
in the future, you can also just use `defaultdict(float)` without needing to manually set a dictionary to 0.